### ISSUE TITLE
[Bug] The parameters section can't be generate under go1.20

### ISSUE THREAD

**Issue by cuizhaoyue** [go-swagger/go-swagger#2913]
## Problem statement

After execute the generate command, the value of body is always "string"
<img width="664" alt="image" src="https://user-images.githubusercontent.com/34601995/225621736-8b6ce0db-cea6-45ce-992a-af3a20f0e250.png">

And then I downgrading the version of Go to 1.19.5 and try to re-generate the swagger files successfully. 

## Environment
swagger version: 0.30.3
go version: 1.20.2
OS:   MacOS

**Comment by rafi0101** [comment]
Yes, same problem

**Comment by ToJuMaLe** [comment]
+1

**Comment by mara-zoratti-pixart** [comment]
same here

**Comment by rafi0101** [comment]
Is there any chance that this problem will be fixed in near future?

**Comment by WinnerOK** [comment]
Possible duplicate of #2897
Yet, no fix found

**Comment by WinnerOK** [comment]
Apparently, the issue I mentioned includes a possible workaround. 
TL;DR:  the issue relates to the platform and Go version which were used to build a `swagger` binary. 
`go-swagger` were not generating `definitions`, when I was using old binary (built for go 1.19) and `$GOROOT` pointing to a version `1.20`

Once I rebuilt `go-swagger` from sources using go 1.20, the `swagger` binary started working properly
Tested `go-swagger` v0.30.3 at M1 Mac

**Comment by fascari-meli** [comment]
Any fix for this issue? 

Still happens on:

Swagger version: 0.30.4
go version: 1.20.4
OS: MacOS

**Comment by Quikr** [comment]
My observation is with the latest version of GoLand (2023.1.2) it is changing the swagger yaml in the comments via its linter. Thus breaking the request body format and responses 
- looking for a fix on the linter

**Comment by rafi0101** [comment]
Seems to be fixed in v0.30.5
https://github.com/go-swagger/go-swagger/releases/tag/v0.30.5

https://github.com/go-swagger/go-swagger/issues/2904

**Comment by Xiphoseer** [comment]
> Seems to be fixed in v0.30.5
> https://github.com/go-swagger/go-swagger/releases/tag/v0.30.5

No it isn't. Experiencing this bug with
```cli
$ go version
go version go1.21.0 windows/amd64
$ swagger_windows_amd64.exe version
version: v0.30.5
commit: ab6a7885723430004f1d7db6395369b6e7f3370b
$ ~\go\bin\swagger.exe version
version: v0.30.5
commit: (unknown, mod sum: "h1:SQ2+xSonWjjoEMOV5tcOnZJVlfyUfCBhGQGArS1b9+U=")
```

**Comment by juburr** [comment]
I can still reproduce in v0.30.5. Works great with Go 1.19, but fails to generate properly with both Go 1.20 and Go 1.21.
